services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -352,10 +352,11 @@ def make(cls, node_uuid: UUID) -> "DynamicSidecarNamesHelper":
         )
 
 
-class SchedulerData(CommonServiceDetails, DynamicSidecarServiceLabels):
+class SchedulerData(CommonServiceDetails):
     # TODO: ANE this object is just the context of the dy-sidecar. Should
     # be called like so and subcontexts for different handlers should
     # also be added. It will make keeping track of env vars more easily
+    DynamicSidecarServiceLabels
```

Comment:
hmm what is this?

Reply:
oohh sorry for that... was a bad commit! 
I wrote that there while discussing with ANE ... the linter caught it but I did not notice it!